File: 2643. Row With Maximum Ones/solution.js
/**
 * @param {number[][]} mat
 * @return {number[]}
 */
var rowAndMaximumOnes = function(mat) {
  let result = [0, 0];
  for (let row = 0; row < mat.length; row++){
    let count = 0;
    for (let column = 0; column < mat[row].length; column++){
      if (mat[row][column] == 1){
        count += 1;
      }
    }
    if (count > result[1]){
      result = [row, count];
    }
  }
  return result;
};

File: 2643. Row With Maximum Ones/solution.java
class Solution {
  public int[] rowAndMaximumOnes(int[][] mat) {
    int[] result = new int[] {0, 0};
    for (int row = 0; row < mat.length; row++){
      int count = 0;
      for (int column = 0; column < mat[row].length; column++){
        if (mat[row][column] == 1){
          count += 1;
        }
      }
      if (count > result[1]){
        result[0] = row;
        result[1] = count;
      }
    }
    return result;
  }
}